deploy_rln_contract.sh

```diff
@@ -14,6 +14,15 @@ if [ ! -d "waku-rlnv2-contract" ]; then
 fi
 
 cd /waku-rlnv2-contract
+git checkout $RLN_CONTRACT_REPO_COMMIT
+# Verify the current commit
+CURRENT_COMMIT=$(git rev-parse HEAD)
```

Comment:
Hm, it seem a good way, but in case one would use master/main/any-branch-name, it would fail as `main` will not match commit hash.
I would just check if $? is zero.

Reply:
I think return value of the `git checkout` is quite reliable. It will be non null if it could not perform the operation.

Reply:
Agreed, I removed the checking in https://github.com/waku-org/waku-simulator/pull/95/commits/a81ae8c122f228809b6c68a83d0e27e8268dc57e

Reply:
I rechecked this. If the value of RLN_CONTRACT_REPO_COMMIT is not a valid commit or branch then it returns an error and exits, but if the value is empty then the command returns 'Your branch is up to date' and continues.
Therefore a check to verify that the variable is set is needed. I'll make another commit for it

Reply:
updated in https://github.com/waku-org/waku-simulator/pull/95/commits/7b8997b3a46759717723cca551ccb15b816291e3